cmd/csi-node-driver-registrar/node_register.go

```diff
@@ -36,6 +37,15 @@ func nodeRegister(
 	// pluginswatcher infrastructure. Node labeling is done by kubelet's csi code.
 	registrar := newRegistrationServer(csiDriverName, *kubeletRegistrationPath, supportedVersions)
 	socketPath := fmt.Sprintf("/registration/%s-reg.sock", csiDriverName)
+
+	ticker := time.NewTicker(5 * time.Second)
+	for {
+		<-ticker.C
+		go periodicallyCheck(socketPath, registrar)
+	}
+}
+
+func periodicallyCheck(socketPath string, registrar registerapi.RegistrationServer) {
```

Comment:
This function can be called something like "reconcileSocket", to be more descriptive of what it does.